CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+<!--
+(c) Copyright IBM Corp. 2017 All Rights Reserved
+
+This code is free software; you can redistribute it and/or modify it
+under the terms of the GNU General Public License version 2 only, as
+published by the Free Software Foundation.
+
+This code is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+version 2 for more details (a copy is included in the LICENSE file that
+accompanied this code).
+
+You should have received a copy of the GNU General Public License version
+2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+-->
+
+## Submitting a contribution to OpenJ9
+
+You can propose contributions by sending pull requests (PRs) through GitHub.
+Following these guidelines will help us merge your pull requests smoothly:
+
+1. Your pull request is an opportunity to explain both what changes you'd like
+   pulled in, but also _why_ you'd like them added. Providing clarity on why
+   you want changes makes it easier to accept, and provides valuable context to
+   review.
+
+2. Follow the commit guidelines found below.
+
+3. We encourage you to open a pull request early, and mark it as "Work In
+   Progress", by prefixing the PR title with "WIP". This allows feedback to
+   start early, and helps create a better end product. Committers will wait
+   until after you've removed the WIP prefix to merge your changes. 
+
+4. Please carefully read and adhere to the legal considerations and
```

Comment:
There are no legal considerations or copyright/license requirements included in this version.

Reply:
I did read carefully enough. :)